layouts/partials/docs/glossary-terms.html

```diff
@@ -1,10 +1,7 @@
 {{ $glossaryBundle := site.GetPage "page" "docs/reference/glossary" }}
 {{- if $glossaryBundle -}}
   {{ $pages := $glossaryBundle.Resources.ByType "page" }}
-  {{- range site.Params.language_alternatives -}}
-    {{- with (where $glossaryBundle.Translations ".Lang" . ) -}}
-    {{ $p := (index . 0) }}{{ $pages = $pages | lang.Merge ($p.Resources.ByType "page") }}{{ end }}
-  {{ end }}
+  {{- range site.Params.language_alternatives -}}{{- with (where $glossaryBundle.Translations ".Lang" . ) -}}{{ $p := (index . 0) }}{{ $pages = $pages | lang.Merge ($p.Resources.ByType "page") }}{{ end }}{{ end }}
```

Comment:
hello @bells17. Thanks for looking into this issue.
Is it possible to fix this issue without taking away the indentation of the template code?
For example,
`{{-` and `-}}` should trim white space, https://golang.org/pkg/text/template#hdr-Text_and_spaces.
I have not tested where the template formatting is needed or if there is a bug.

{{- $pages = $pages | lang.Merge ($p.Resources.ByType "page") -}} or possibly another line.

Reply:
Hi @kbhawkey, thank you for your comment.
I improved this code, Please review it again.